packages/contracts-bedrock/src/L1/L2OutputOracle.sol

```diff
@@ -1,11 +1,16 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.15;
 
+// Contracts
 import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
-import { ISemver } from "src/universal/interfaces/ISemver.sol";
+
+// Libraries
 import { Types } from "src/libraries/Types.sol";
 import { Constants } from "src/libraries/Constants.sol";
 
+// Interfaces
+import { ISemver } from "src/universal/interfaces/ISemver.sol";
```

Comment:
In the foundry config we can set `sort_imports = true` to avoid manual sorting, would probably be a big diff so worthy of it's own PR